### DISCUSSION TITLE
A Fundamental Redesign of the PlebWeb

### DISCUSSION THREAD

**Discussion by HyperCrowd** [PsySecGroup/foundation#15] [Ideas]
With the rapid advancement of AI, it's time to take a closer look at open source development in the PlebWeb. In the good old days, it was a user-friendly app ecosystem where everyday folks could chat and connect without worrying too much about security.

But now, AI has set its sights on the PlebWeb, making it the prime target. And how are the custodians of the PlebWeb, like Reddit, Twitter, and Facebook, responding? Well, they're building walls and playing defense to protect themselves.

So here's the deal: we need to give the PlebWeb a serious makeover. We've got to rethink what it's all about, what problems it solves, how it operates, and why it needs to shield itself from AI threats.

Let's dig into the core principles of the PlebWeb and figure out what makes it tick.

**Comment by dscnode** [comment]
Throwing user indexing (_UIDX_) in here:
[https://dynode-io.ipns.dweb.link/uidx.html](https://dynode-io.ipns.dweb.link/uidx.html)
